config/config.default.yaml

```diff
@@ -257,6 +257,13 @@ conventional:
   nuclear:
     p_max_pu: "data/nuclear_p_max_pu.csv" # float of file name
 
+TYNDP2020:
+  include: false
+  allowed_statuses: ['in_permitting', 'under_construction']  # 'planned_not_yet_permitting', 'under_consideration'
```

Comment:
maybe you have to add these lines also in the test.
CI is failing now because some indexing or similar pandas error

Reply:
no thats not the problem. the problam is somehow related to the new pandas version and trying to add data to an empty dataframe